platform/android/java/lib/res/mipmap-anydpi-v26/icon.xml

```diff
@@ -2,5 +2,4 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Let's add a comment to warn that the contents of this file should always match the constant defined in `export_plugin.cpp`